### PR TITLE
Alex

### PR DESCRIPTION
**Pull request by Aleksei12942**

### CODE REVIEW

File: app/assets/javascripts/trainings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+[{"title":"Тренировка с клиентом 1","start":"2018-08-10 10:35:00","price":20,"description":"Описание","status":"planned"},{"title":"Тренировка с клиентом 2","start":"2018-08-09 19:05:00","price":50,"description":"Описание","status":"planned"},{"title":"Тренировка с клиентом 3","start":"2018-08-11 12:45:00","price":80,"description":"Описание","status":"planned"},{"title":"Тренировка с клиентом 4","start":"2018-08-11 12:55:00","price":100,"description":"Описание","status":"planned"}]
```
Comment (LikaLex) [inline app/assets/javascripts/trainings.json]: Delete it, please

File: app/models/exercise.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Exercise < ApplicationRecord
+  has_many :exercise_types
```
Comment (LikaLex) [inline app/models/exercise.rb]: maybe it should belong to exercise type?

File: app/controllers/kits_controller.rb [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  before_action :find_kit, only: [:show, :edit, :update, :destroy]
+  before_action :authenticate_user!
+  protect_from_forgery with: :null_session
+  skip_before_action :verify_authenticity_token
+  @@div_id = 1
+
+  def index
+    @kit = Kit.all.map do |kit|
+      {
+          :client => (Client.find_by(id: kit.client_id)).first_name + ' ' + (Client.find_by(id: kit.client_id)).second_name,
```
Comment (LikaLex) [inline app/controllers/kits_controller.rb]: рубокоп на старые хэши ругаться будет

File: app/controllers/exercise_types_controller.rb [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  def index
+    @exercise_type = ExerciseType.all
+  end
+
+  def new
+    @exercise_type = ExerciseType.new
+  end
+
+  def create
+    #require "pry"; binding.pry
```
Comment (Equinox-01) [inline app/controllers/exercise_types_controller.rb]: Remove comment from file

File: app/controllers/exercises_controller.rb [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+    @exercise = Exercise.new(:exercise_type_id => 1, :kit_id => kit_id, :user_id => current_user.id, :repeats => 1, :approach => 1)
+    if @exercise.save
+      redirect_to edit_exercise_path(@exercise)
+    else
+      render 'new'
+    end
+  end
+
+  def edit
+    @count = @@count
```
Comment (Equinox-01) [inline app/controllers/exercises_controller.rb]: Very bad

**Pull request merged**